util/db/cluster.go

```diff
@@ -232,6 +232,34 @@ func (db *db) GetProjectClusters(ctx context.Context, project string) ([]*appv1.
 	return res, nil
 }
 
+func (db *db) GetClusterServersByName(ctx context.Context, name string) ([]string, error) {
+	informer, err := db.settingsMgr.GetSecretsInformer()
+	if err != nil {
+		return nil, err
+	}
+
+	// if local cluster name is not overridden and specified name is local cluster name, return local cluster server
+	localClusterSecrets, err := informer.GetIndexer().ByIndex(settings.ByClusterURLIndexer, appv1.KubernetesInternalAPIServerAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(localClusterSecrets) == 0 && name == string(localClusterSecrets[0].(*apiv1.Secret).Data["name"]) {
```

Comment:
Should this be `en(localClusterSecrets) > 0`?

Reply:
ops, good catch! Fixed.

Reply:
Unit test also catch it so I had to modify implementation only

Reply:
👍